themes/classic/templates/checkout/_partials/cart-voucher.tpl

```diff
@@ -77,7 +72,11 @@
           {/foreach}
           </ul>
         {/if}
+          <a class="collapse-button promo-code-button cancel-promo" role="button" data-toggle="collapse" data-target="#promo-code" aria-expanded="true" aria-controls="promo-code">
+            {l s='Close' d='Shop.Theme.Checkout'}
+          </a>
+        </div>
       </div>
     </div>
   {/block}
-{/if}
+{/if}
```

Comment:
You remove the line at the end of the file.

use `git add -p` to wath what you change and be sure you want to add this patch :wink:

Reply:
What are you talking about? Nothing was removed 🤔

Reply:
```suggestion
{/if}

```
There was a trailing line at the end of the file

Reply:
I added the line